.ci/setup_env_ubuntu.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir=$(dirname "$0")
+source "/etc/os-release"
+source "${cidir}/lib.sh"
+get_cc_versions
+
+arch=$(arch)
+
+if [ "$arch" = x86_64 ]; then
+	if grep -q "N" /sys/module/kvm_intel/parameters/nested; then
+		echo "enable Nested Virtualization"
+		sudo modprobe -r kvm_intel
+		sudo modprobe kvm_intel nested=1
+	fi
+else
+	die "Unsupported architecture: $arch"
+fi
+
+echo "Update apt repositories"
+sudo -E apt update
+
+echo "Install chronic"
+sudo -E apt install -y moreutils
+
+echo "Install kata containers dependencies"
+chronic sudo -E apt install -y libtool automake autotools-dev autoconf bc alien libpixman-1-dev coreutils
+
+echo "Install qemu-lite binary"
+"${cidir}/install_qemu_lite.sh" "${qemu_lite_clear_release}" "${qemu_lite_sha}" "$ID"
+
+echo "Install kata-containers image"
+"${cidir}/install_kata_image.sh"
```

Comment:
When we run this script, we already need docker, and if it is not installed, this will fail. We should install docker using the `manage_ctr_mgr.sh` script before calling `install_kata_image.sh`.